test/workbox-precaching/sw/test-PrecacheController.mjs

```diff
@@ -486,6 +486,56 @@ describe(`PrecacheController`, function() {
       const {request} = fetchWrapper.fetch.args[0][0];
       expect(request.credentials).to.eql('same-origin');
     });
+
+    it(`it should use the integrity value when making requests`, async function() {
+      const fetchSpy = sandbox.spy(fetchWrapper, 'fetch');
+
+      const precacheController = new PrecacheController();
+      const cacheList = [
+        {url: '/first'},
+        {url: '/second', integrity: 'sha256-second'},
+      ];
+      precacheController.addToCacheList(cacheList);
+
+      await precacheController.install();
+
+      expect(fetchSpy.calledTwice).to.be.true;
+      expect(fetchSpy.firstCall.args[0].request.integrity).to.eql('');
+      expect(fetchSpy.secondCall.args[0].request.integrity).to.eql('sha256-second');
+    });
+
+    it(`it should use the last-specified integrity value when making requests`, async function() {
+      const fetchSpy = sandbox.spy(fetchWrapper, 'fetch');
+
+      const precacheController = new PrecacheController();
+      const cacheList = [
+        {url: '/test', integrity: 'sha256-one'},
+        {url: '/test', integrity: 'sha256-two'},
```

Comment:
As mentioned above, I would expect this to be an error. What's the rationale for supporting this use case?

Reply:
I've changed the test to expect this to fail.